playwright.config.ts

```diff
@@ -13,6 +13,7 @@ import { defineConfig, devices } from "@playwright/test";
  */
 export default defineConfig({
   testDir: "./",
+  testMatch: "**/*.spec.ts",
```

Comment:
Manual test [fails](https://github.com/Voomlz/voomlz.github.io/actions/runs/13741845618/job/38432019663?pr=47) because the workflow at `master` has playwright trying to read the .test.ts files. I've ignored them in this config ^

Local runs pass:

```
 Test Files  4 passed (4)
      Tests  33 passed | 1 skipped (34)
   Start at  08:19:26
   Duration  230ms (transform 107ms, setup 0ms, collect 215ms, tests 17ms, environment 0ms, prepare 164ms)

 PASS 

npx playwright test


Running 40 tests using 5 workers
  40 passed (21.0s)
```